<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="generator" content="Jmol, see http://www.jmol.org">
  <meta name="author" content="Sylvia Burris">
  <meta name="keywords" content="generated_by_Jmol_Webexport">
  <title>Lengsin</title>
  <style type="text/css">
	h1 { font-size:1.5em; text-align:center; }
	.section { clear:both; padding:0.6em 0; }
	.sceneWrapper {
		border:1px solid gray;
		border-radius:1ex;
		margin:0.3em 1ex; 
		font-size:0.9em; 
	}
	.sceneWrapper td {
		padding:0.5ex;
	}
  .JmolDiv { 
		padding:0.3em 0.5ex; 
		font-size:0.9em; 
		background-repeat:no-repeat; 
		background-position:center;
  }
  .floatRight { float:right; }
  .floatLeft { float:left; }
  .update-info {
		clear:both;
		padding-top:1em;
		text-align:right;
		font-size:0.75em;
		font-style:italic;
  }
	.update-info + .update-info {
		padding-top:0.2em;
		margin-bottom:1em;
	}
	.widgetItemPopin { /* each widget in a PopIn page goes in a div with this class */
		 margin:0.2em 0;
	}
  </style>  

  <!-- Below is used locally but not on server. May generate warnings. -->
  <script type="text/javascript" src="http://binf.gmu.edu/sburris3/edu/jmol/jsmol/JSmol.min.js"></script>
  <script src="support.js" type="text/javascript"></script>
  
  
  <!-- Below is used on server, but not locally. May generate warnings.-->
  <script type="text/javascript" src="http://binf.gmu.edu/sburris3/jmol/jsmol/JSmol.min.js"></script>
  <script type="text/javascript">
  var jmolApplet; //our viewer
  var jmolInfo; //setup parameters
  ;(function() {

// Developers: The debugCode flag is checked in j2s/java/core.z.js, 
// and, if TRUE, skips loading the core methods, forcing those
// to be read from their individual directories. Set this
// true if you want to do some code debugging by inserting
// System.out.println, document.title, or alert commands
// anywhere in the Java or Jmol code.

var s = document.location.search;
Jmol.debugCode = (s.indexOf("debugcode") >= 0);

jmol_isReady = function(jmolApplet) {
}		

jmolInfo = { //default values
	width: "100%",
	height: "100%",
	debug: false,
	color: "white",
	addSelectionOptions: false,
	use: "HTML5",
	coverImage: "@APPLETNAME0@.png",        // initial image instead of applet
	coverScript: "",	// special script for click of cover image (otherwise equal to script)
	deferApplet: true,                  // wait to load applet until click
	deferUncover: true,                 // wait to uncover applet until script completed
		//The paths below assume your server is set up with standard JSmol directory.  If not
		//they will need modification for the page to work.
	jarPath: "http://binf.gmu.edu/sburris3/jmol/jsmol/java", //path to applet .jar files on server.
	j2sPath: "http://binf.gmu.edu/sburris3/jmol/jsmol/j2s",//path to javascript version.
	serverURL: "http://binf.gmu.edu/sburris3/jmol/jsmol/php/jsmol.php", //if your own server does not allow php, you can change to http://chemapps.stolaf.edu/jmol/jsmol/php/jsmol.php
	makeLiveImg:"http://binf.gmu.edu/sburris3/jmol/jsmol/j2s/img/play_make_live.jpg",//path to activate 3-D image.
	jarFile: "JmolAppletSigned0.jar",
	isSigned: true,
	//disableJ2SLoadMonitor: true,
	disableInitialConsole: true,
	readyFunction: jmol_isReady,
	script: ""
}

// ?_USE=JAVA   ?_USE=SIGNED   ?_USE=HTML5
   if (s.indexOf("USE=") >= 0)
     jmolInfo.use = s.split("USE=")[1].split("&")[0]
   else if (s.indexOf("JAVA") >= 0)
     jmolInfo.use = "JAVA"
   else if (s.indexOf("IMAGE") >= 0)
     jmolInfo.use = "IMAGE"
   else if (s.indexOf("NOWEBGL") >= 0)
     jmolInfo.use = "JAVA IMAGE"
   else if (s.indexOf("WEBGL") >= 0)
     jmolInfo.use = "WEBGL HTML5"
   if (s.indexOf("NOWEBGL") >= 0)
     jmolInfo.use = use.replace(/WEBGL/,"")
   var useSignedApplet = (s.indexOf("SIGNED") >= 0);
   if(useSignedApplet && jmolInfo.use == "HTML5") jmolInfo.use = "JAVA";

   var protocol = window.location.protocol.toLowerCase();
   if (protocol == "file:") { 
    jmolInfo.jarPath = "http://binf.gmu.edu/sburris3/edu/jmol/jsmol/java";
    jmolInfo.j2sPath = "http://binf.gmu.edu/sburris3/edu/jmol/jsmol/j2s";
    jmolInfo.serverURL = "http://chemapps.stolaf.edu/jmol/jsmol/php/jsmol.php"; //you can change to your own path
    jmolInfo.makeLiveImg = "http://binf.gmu.edu/sburris3/edu/jmol/jsmol/j2s/img/play_make_live.jpg";
    jmolInfo.jarFile = "JmolAppletSigned.jar";
    jmolInfo.isSigned= "true";
  }
})();

$(document).ready(function(){//set up each of the applet locations
	   var jmolInfo0=jmolInfo;
   jmolInfo0.coverImage="Lengsin.png";
   jmolInfo0.coverScript="javascript revealPopinWidgets(0);";
   jmolInfo0.script="load Lengsin.spt";
   $("#Jmol0").html(Jmol.getAppletHtml("jmolApplet0",jmolInfo0));

});
  </script>
</head>
<body>
    <div id="introdiv">
      <h1 id="titlediv">
Protein Modelling by Sylvia Burris
      </h1>
I selected a sequence of a human enzyme from the NCBI database. I looked up a protein sequence of enzymes and used the input (enzyme) AND "Homo sapiens"[porgn:__txid9606]  to specify only enzymes from homo sapiens. Using additional filters on the website, I put a filter for sequences to be between 150- 250 residues and excluded enzymes that were also available in the PDB database or had identical sequences in PDB.  
I then checked the query sequences in the PDB database to make sure that the enzyme had a homolog that had a sequence similarity between 30% and 60%. 
lengsin isoform b [Homo sapiens] (NCBI Reference Sequence: NP_001137412.1) was selected. It has an amino acid residue of 208 aa. It has one homolog in PDB with a sequence similarity of 52%, Lengsin (PDB :2J9I).     </div>

    

    <div id="section_Lengsin" class="section">
	<table id="scene_Lengsin" class="sceneWrapper floatRight" style="width:300px;">
		<tr>
			<td id="leftJmolCntl0" style="visibility:hidden"></td>
			<td style="width:300px;">
			<div id="Jmol0" class="JmolDiv" style="width:300px; height:300px; background-image:url(Lengsin.png)">
				Jmol0 will appear here.
			</div>
			</td>
			<td id="rightJmolCntl0" style="visibility:hidden"></td>
		</tr>
		<tr>
			<td colspan="3">     
				CLICK IMAGE TO ACTIVATE 3D <br/> lengsin isoform b [Homo sapiens].		
			</td>
		</tr>
	</table>
	<div id="text_Lengsin">
Three-dimensional structure of lengsin isoform b was predicted using a swiss-model. The Swiss-model generated two models using 50 templates. 
I used the structure validation tool Save v6.0 https://saves.mbi.ucla.edu/ to verify the model obtained above. The overall quality of the of the model is 78.5124%
This model fails the verification because 66.67% of the residues have averaged 3D-1D score >= 0.2 and fewer than 80% of the amino acids have scored >= 0.2 in the 3D/1D profile.
A tyrosine residue was introduced between two Alanine residues at position 189 and 190. I chose a tyrosine residue because I wanted to see the effect of a hydrophobic residue on the alpha helix. 
There was a slight change to the structure, the alpha helix at the end of the protein structure was shortened by the modeling. The modeled structure had a shorter amino acid sequence. 	</div>

</div>



    <div id="creationdiv" class="update-info">
      Page skeleton and JavaScript generated by the Export to Web module of  <a href="http://jmol.sourceforge.net">Jmol 14.32.81  2022-11-22 15:50</a>  on Dec 4, 2022.
    </div>
    <div id="authordiv" class="update-info">
      Based on a template by A. Herr&#x00E1;ez and J. Gutow
    </div>
    <div id="logdiv" style="display:none">
      <pre>
Using directory C:/Users/sylvi/OneDrive/Documents/Lengsin
  adding support.js
  ...jmolApplet0
      ...adding Lengsin.png
      ...copying
C:/Users/sylvi/Downloads/model_01.pdb
         to
C:/Users/sylvi/OneDrive/Documents/Lengsin/model_01.pdb
      ...adding Lengsin.spt

</pre>

    </div>
</body>
</html>
